client/components/Main.jsx

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
 import {Link} from 'react-router-dom'
 import Map from './map.jsx';
-let loggedIn;
 //loggedIn will be set based off of if they have a cookie or not
 const Main = ()=>{
+  let loggedIn = true;
   let test = <a href='./google/auth'><button id="signin">Sign in with Google</button></a>;
   //the button will change from sign in to a go to profile button based on whether or not they are signed in
   if(loggedIn===false){
-    test = <button onClick={location.assign('/profile')}id='profile'>Go to profile</button>
+    test = <button onClick={()=>{location.assign('/bathroom')}}id='bathroomButton'>Go to profile</button>
```

Comment:
I don't think you need brackets around location.assign, it's already being evaluated as JS